README.md

```diff
@@ -12,3 +12,29 @@ make it easier for developers to build AI-applications that integrate with Table
 ## Official Documentation
 
 https://tableau.github.io/tableau-mcp/
+
+## Quick Start
```

Comment:
I'm adding this to the root README so it shows up on npmjs and users won't have to go digging in the docs site.